{"is_answered": true, "view_count": 39, "tags": ["python", "python-2.7", "pandas", "indexing", "dataframe"], "last_activity_date": 1467319973, "answer_count": 2, "creation_date": 1467319668, "score": 1, "link": "http://stackoverflow.com/questions/38132976/a-value-is-trying-to-be-set-on-a-copy-of-a-slice-from-a-dataframe-even-when-l", "accepted_answer_id": 38133027, "owner": {"user_id": 2579896, "profile_image": "https://www.gravatar.com/avatar/34693697dc5c681bfa0d144b5c7a6258?s=128&d=identicon&r=PG&f=1", "user_type": "registered", "reputation": 697, "link": "http://stackoverflow.com/users/2579896/juan-david", "accept_rate": 79, "display_name": "Juan David"}, "title": "&quot;A value is trying to be set on a copy of a slice from a DataFrame&quot; even when .loc is used", "question_id": 38132976}